src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -41,21 +42,48 @@ internal static class IntellisenseQuickInfoBuilder
                 firstLineElements.Add(new ImageElement(warningGlyph.GetImageId()));
             }
 
+            var elements = new List<object>();
             var descSection = quickInfoItem.Sections.FirstOrDefault(s => s.Kind == QuickInfoSectionKinds.Description);
             if (descSection != null)
             {
-                firstLineElements.Add(BuildClassifiedTextElement(descSection));
+                var isFirstElement = true;
+                foreach (var element in BuildClassifiedTextElements(descSection))
+                {
+                    if (isFirstElement)
+                    {
+                        isFirstElement = false;
+                        firstLineElements.Add(element);
+                    }
+                    else
+                    {
+                        elements.Add(element);
+                    }
+                }
             }
 
-            var elements = new List<object>
+            elements.Insert(0, new ContainerElement(ContainerElementStyle.Wrapped, firstLineElements));
```

Comment:
this container element still needed if description is null? or desSection is empty?

Reply:
I have no idea how to produce such an element.